packages/management/addons-manager/README.md

```diff
@@ -0,0 +1,19 @@
+# addons-manager Package
+
+Addon-manager generates App CRs for addons based on TKR/BOM, KubeadmControlPlane and CLI inputs provided during cluster creation.
```

Comment:
nit: `Addon-manager generates addon secrets which contain the configurations for addons based on xxx` ?

Reply:
thanks for the correction! updated README